packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -112,6 +112,16 @@
   display: flex;
   width: 100%;
   box-sizing: border-box;
+  &:before {
```

Comment:
Sounds like `::after` is already in use by the icons. We could update the mixin to create either a before or after, but this does feel like scope creep. The current changes look fine to me and we can always revisit this in the future. 😄